tests/job/test_script.py

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import unittest
+import os
+from pyiron_base.project.generic import Project
+
+
+class TestScriptJob(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.file_location = os.path.dirname(os.path.abspath(__file__)).replace(
+            "\\", "/"
+        )
+        cls.project = Project(os.path.join(cls.file_location, "test_sriptjob"))
+
+    @classmethod
+    def tearDownClass(cls):
+        file_location = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Codacy appears correct to complain about this, indeed it doesn't appear to be used.